config/locales/en.yml

```
@@ -192,7 +192,7 @@ en:
             <p>We don't provide a certificate for course completion. Employers will be much more impressed with your amazing personal portfolio of projects, many of which you'll hopefully have built with inspiration from The Odin Project.</p>
         - question: 'Do you have a Code of Conduct?'
           answer: |
-            <p>You can read our code of conduct <a href='https://github.com/TheOdinProject/theodinproject/blob/main/doc/code_of_conduct.md'>here</a></p>
+            <p>You can read our code of conduct <a href='https://github.com/TheOdinProject/.github/blob/ee395d922d7622f1420659a72d193e0c6b8d8e7c/CODE_OF_CONDUCT.md'>here</a></p>
```

Could we change the link to the version on the main branch please? It will look a bit cleaner.

```suggestion
            <p>You can read our code of conduct <a href='https://github.com/TheOdinProject/.github/blob/main/CODE_OF_CONDUCT.md'>here</a></p>
```

---

@ChargrilledChook on it!

---

@ChargrilledChook and thanks for the feedback!

---

@ChargrilledChook done!